<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScene</class>
 <widget class="QMainWindow" name="MainScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>588</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>588</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>588</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>翻金币游戏</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionnewButton"/>
    <addaction name="actionnewButton2"/>
    <addaction name="actionnewButton3"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionnewButton">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionnewButton2">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="actionnewButton3">
   <property name="text">
    <string>关于作者</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
